development/PersonOntology.owl

```diff
@@ -424,7 +347,25 @@
         <rdfs:label xml:lang="en">has given name</rdfs:label>
     </owl:DatatypeProperty>
     
+    
+    <!-- https://w3id.org/enigma#hasHomePage -->
 
+    <owl:DatatypeProperty rdf:about="https://w3id.org/enigma#hasHomePage">
+        <rdfs:subPropertyOf rdf:resource="http://xmlns.com/foaf/0.1/homepage"/>
```

Comment:
You cannot make a data property subproperty of an object property. Delete that hasHomePage is a subproperty of foaf:homepage